runtime/src/proofs.rs

```diff
@@ -6,8 +6,8 @@ use sp_std::vec::Vec;
 #[cfg_attr(not(feature = "std"), derive(sp_runtime::RuntimeDebug))]
 #[cfg_attr(feature = "std", derive(Debug))]
 pub struct Proof {
-    leaf_hash: H256,
-    sorted_hashes: Vec<H256>,
+    pub leaf_hash: H256,
+    pub sorted_hashes: Vec<H256>,
```

Comment:
why do we need to make `sorted_hashes` public?

Reply:
The same as last comment. It is strange to just set one member as public to me. I can't see any side effect here.